src/main/java/uk/ac/cam/cl/dtg/segue/api/AuthenticationFacade.java

```diff
@@ -98,6 +99,42 @@ public AuthenticationFacade(final PropertiesLoader properties, final UserAccount
         this.misuseMonitor = misuseMonitor;
     }
 
+    /**
+     * Get authentication provider information for the current user
+     *
+     * @param request
+     *            - the http request containing session information of the user currently logged in
+     * @param userId - the id of the user of interest
+     * @return a user authentication settings object showing information about how a given user can authenticate
+     */
+    @GET
+    @Path("/user_authentication_settings/{user_id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "The current users authentication settings, e.g. linked accounts and whether they have segue or not")
+    public final Response getCurrentUserAuthorisationSettings(@Context final HttpServletRequest request, @PathParam("user_id") Long userId ) {
+        try {
+            RegisteredUserDTO currentRegisteredUser = this.userManager.getCurrentRegisteredUser(request);
+
+            if (!userId.equals(currentRegisteredUser.getId()) && !currentRegisteredUser.getRole().equals(Role.ADMIN)) {
```

Comment:
There's also an `isUserAnAdmin(userManager, request)` facade method, would it be worth using that instead?

Reply:
Ah yes I originally put that, then realised that I already had to extract the register user and that method does it again... I may make a new facade method that will work if you already have the user.